### PR TITLE
added service mesh patterns

### LINKED ISSUE #3019: [Handbook] Frontend Projects: Add service-mesh-patterns to the list
#### Current Behavior
The list of frontend projects (https://layer5.io/community/handbook/repository-overview) in the community handbook is missing the new service-mesh-patterns project.

#### Desired Behavior
Add Service Mesh Patterns to the list of projects. 

- Site: https://service-mesh-patterns.github.io/service-mesh-patterns/
- GitHub: https://github.com/service-mesh-patterns/service-mesh-patterns
- Logo: Find here - https://layer5.io/company/brand

---
<h4> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by manikmmalhotra**
Signed-off-by: manik malhotra <manikmalhotra2113@gmail.com>

**Description**
added service mesh patterns
Screenshot from 2022-08-04 02-56-10

This PR fixes #3019 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, @manikmmalhotra. Any chance that you can rearrange these into alphabetical order?

**Comment by l5io** [comment]
🚀 Preview for commit 00649cfdb35ef7b6647158ed0dae92e1e513793b at: https://62eaeaff4761cf1b79ffb1ea--layer5.netlify.app

**Comment by manikmmalhotra** [comment]
@leecalcote sure

**Comment by l5io** [comment]
🚀 Preview for commit 7ed4b71bb67d4853333fa207b99465fed8d9e0e4 at: https://62eb5a2de0ce2f45f803cb05--layer5.netlify.app

**Review by karnatisrinivas — COMMENTED** [review]
Nice work @manikmmalhotra, Looks good 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable

**Comment by leecalcote** [comment]
Thank you, gents.

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -76,13 +61,35 @@ export const repo_data = [
...
         image: five,
         site: "https://discuss.layer5.io",
         language: "Discourse",
         repository: " https://discuss.layer5.io",
       },
+      {
+        project: "Service Mesh Patterns",
+        image: patterns,
+        site: "https://service-mesh-patterns.github.io/service-mesh-patterns/",
+        language: "JavaScript",
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:82]:
```suggestion
        language: "Gatsby",
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**